### PR TITLE
44 small changes

### PR DESCRIPTION
**Pull request by dikastes**

### CODE REVIEW

**Comment by dikastes** [comment]
Please check especially the changes in the BibEntryMapping.

File: Classes/Processing/BibElasticMapping.php [hunk 1]
```diff
@@ -15,22 +15,28 @@ public static function getMappingParams(string $index): array
...
                         'archiveLocation' => [ 'type' => 'text', 'fields' => [ 'keyword' => [ 'type' => 'keyword', 'ignore_above' => 256 ] ] ],
-                        'itemType' => [ 'type' => 'keyword'],
                         'journalTitle'  => [ 'type' => 'keyword'],
                         'creators' => [
                             'type' => 'nested',
                             'properties' => [
                                 'creatorType' => [
-                                    'type' => 'keyword'
+                                    'type' => 'text',
+                                    'fields' => [
```
Comment (thomas-sc) [inline Classes/Processing/BibElasticMapping.php:35]: change creatorType from type keyword to text and creatorType.keyword means you will also textsearch in creatorType?
(the key for the facette in serachresults will probably change from creatorType to creatorType.keyword)

File: Classes/Processing/BibElasticMapping.php [hunk 2]
```diff
@@ -15,22 +15,28 @@ public static function getMappingParams(string $index): array
...
                     'dynamic' => false,
                     'properties' => [
+                        'itemType' => [ 'type' => 'text', 'fields' => [ 'keyword' => [ 'type' => 'keyword', 'ignore_above' => 256 ] ] ],
                         'version' => [ 'type' => 'long' ],
                         'title' => [ 'type' => 'text'],
                         'university' => [ 'type' => 'text'],
                         'bookTitle' => [ 'type' => 'text'],
                         'series' => [ 'type' => 'text', 'fields' => [ 'keyword' => [ 'type' => 'keyword', 'ignore_above' => 256 ] ] ],
                         'publicationTitle' => [ 'type' => 'text', 'fields' => [ 'keyword' => [ 'type' => 'keyword', 'ignore_above' => 256 ] ] ],
+                        'language' => [ 'type' => 'text', 'fields' => [ 'keyword' => [ 'type' => 'keyword', 'ignore_above' => 256 ] ] ],
```
Comment (thomas-sc) [inline Classes/Processing/BibElasticMapping.php:25]: maybe simple [ 'type' => 'keyword'] would be sufficient

**Review by thomas-sc — DISMISSED** [review]
see small comments in BibEntryMapping

**Pull request merged**